components/progress-bar/src/CircularProgressBar.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { CSSProperties, useEffect, useState } from "react";
+import { useProgress } from "@locoworks/reusejs-toolkit-react-hooks";
+import { twMerge } from "tailwind-merge";
+
+interface ProgressBarInterface {
```

Comment:
Change this name to CircularProgressInterface

Reply:
name changes